content/03.use-cases-a.md

```diff
@@ -6,35 +6,34 @@
-Through version control, the history of code and data from projects in ecology and evolution can be tracked accessible to future laboratory members and collaborators [@doi:10.1038/s41559-017-0160].
-Repositories and organizations can have designated data and code owners [or more appropriate, stewards; see @doi:10.1890/ES14-00402.1, @url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners], who can also change through time allowing for the transition of code between research cohorts (see also ["Organizing and managing teams"](#organizations)).
-Other project collaborators can contribute to repository design and development, and their active involvement can both aid authors ability to act as guarantors of the project, and the clarity and reproducibility of the project for future users.
-In (Figure @fig:github-diagram), we highlight several elements of good repository structure, and the various ways that contributors may interact with them.
+GitHub can facilitate project continuity in research by making code and data handover between users easier [@doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7].
+Through version control, the history of code and data from projects in ecology and evolution becomes accessible to future laboratory members and collaborators [@doi:10.1038/s41559-017-0160].
+Repositories and organizations can have designated data and code owners [or more appropriate, "data stewards"; see @doi:10.1890/ES14-00402.1, @url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners], who can also change through time, allowing for the transition of code between research cohorts (see also ["Organizing and managing teams"](#organizations)).
+Other project collaborators can contribute to repository design and development, and their active involvement can both aid authors ability to act as guarantors, and the clarity and reproducibility of the project for future users.
+In (Figure @fig:github-diagram), we highlight several elements of recommended repository structure, and the various ways that contributors may interact with them.
 
-GitHub also allows users to describe and store information about software and code dependencies, ensuring that code can run using the same version of software as when it was initially developed.
-This can be achieved through the access of GitHub repositories containing the necessary releases of such software, or through the use of automated Github Actions with specific software versions.
+Software compatibility during the analysis and reanalysis of project data can be ensured by storing information about software dependencies and their versions within the same project repository.
+With more advanced practices, one can remotely install and execute scripts using specific versions of software within GitHub's project automation tools, GitHub Actions (see below).
```

Comment:
This sentence and the one above might be too technical for this manuscript? I'm not sure. We're talking indirectly about things like `conda`, `pipenv` and `requirements.txt` files. Would it be helpful to call them out directly or just remove these two sentences because this type of work falls under a more advanced use case.

Reply:
I agree—if we are talking about tracking versions of dependencies, then that is not really a GitHub-specific thing (although it is very useful for getting GitHub actions working)